src/main/java/com/contentgrid/gateway/security/jwt/issuer/SignedJwtIssuer.java

```diff
@@ -47,13 +47,7 @@ public Mono<OAuth2Token> issueSubstitutionToken(ServerWebExchange exchange) {
                     }
                     return Mono.empty();
                 })
-                .flatMap(claims -> {
-                    try {
-                        return Mono.just(claimsSigner.sign(claims));
-                    } catch (JOSEException e) {
-                        return Mono.error(e);
-                    }
-                })
+                .flatMap(claims -> Mono.just(claimsSigner.sign(claims)))
```

Comment:
```suggestion
                .map(claimsSigner::sign)
```

There is no reason anymore to make this a flatMap